ios/RNPlayerViewManager.m

```diff
@@ -21,6 +21,7 @@ @interface RCT_EXTERN_REMAP_MODULE(NativePlayerView, RNPlayerViewManager, RCTVie
 RCT_EXPORT_VIEW_PROPERTY(onStallStarted, RCTBubblingEventBlock)
 RCT_EXPORT_VIEW_PROPERTY(onStallEnded, RCTBubblingEventBlock)
 RCT_EXPORT_VIEW_PROPERTY(onTimeChanged, RCTBubblingEventBlock)
+RCT_EXPORT_VIEW_PROPERTY(onPlaybackSpeedChanged, RCTBubblingEventBlock)
```

Comment:
same here, why this location?

Reply:
address in 1fa91bb913b00693336059784af698ad64254879